### PR TITLE
add info on forward slash escape char

### PR DESCRIPTION
**Pull request by chipzoller**

### CODE REVIEW

File: content/en/docs/Writing policies/mutate.md [hunk 1]
```diff
@@ -163,6 +163,14 @@ mutate:
       value: {"key":"networkzone","operator":"Equal","value":"dmz","effect":"NoSchedule"}
 ```
 
+When needing to apply a mutation containing a special character like a forward slash (`/`) such as when adding an annotation, use `~1` as the escape character. The following adds an annotation with the key of `config.linkerd.io/skip-outbound-ports` with the value of `"8200"`.
+
```
Comment (JimBugwadia) [inline content/en/docs/Writing policies/mutate.md:167]: The other character that needs to be escaped is "~".

Suggest we phrase as,

JSON Patch uses [JSON Pointer](http://jsonpatch.com/#json-pointer) to reference keys, and keys with `~` and `/` characters need to be escaped with ~0 and ~1 respectively. 

For example, the following...

**Review by JimBugwadia — CHANGES_REQUESTED** [review]
Thanks! Can we also add escape instructions for `~`?

File: content/en/docs/Writing policies/mutate.md (hunk 1 above)
Comment (chipzoller) [reply content/en/docs/Writing policies/mutate.md:167]: Sure, just added.

**Review by JimBugwadia — APPROVED** [review]

**Comment by JimBugwadia** [comment]
Thanks @chipzoller !

**Pull request merged**